### ISSUE TITLE
Steam Input does not work in gaming mode on steam deck after lutris update.

### ISSUE THREAD

**Issue by midarikishima** [lutris/lutris#5780]
### Bug description

After updating to the latest lutris version on the steam deck steam input does not work in the steam gaming mode, the only input that will work is an external keyboard. Any other controller is not recognised. 

### How to Reproduce

1. Update lutris on steam deck.
2. Launch Gaming Mode
3. Open any game that has been added to steam through lutris.
4. Thats it steam input dosent work.

### Expected behavior

like from the previous lutris version steam input should work in gaming mode with no issues.

### Log output

```shell
[INFO:2024-12-10 01:01:51,114:application]: Starting Lutris 0.5.18
[INFO:2024-12-10 01:01:51,158:startup]: "card0" is AMD Custom GPU 0932 (1002:1435 1002:0123 amdgpu) Driver 24.2.7
```

### System Information

```shell
Vulkan support: YES
Esync support: YES
Fsync support: YES
Wine installed: YES
Gamescope: NO
Mangohud: NO
Gamemode: YES
Steam: YES
In Flatpak: YES
[System]
OS: GNOME 47 Flatpak runtime
Arch: x86_64
Kernel: 6.5.0-valve22-1-neptune-65-g9a338ed8a75e
Lutris Version: 0.5.18
Desktop: KDE
Display Server: x11
[CPU]
Vendor: AuthenticAMD
Model: AMD Custom APU 0932
Physical cores: 4
Logical cores: 8
[Memory]
RAM: 14.5 GB
Swap: 8.2 GB
[Graphics]
Vendor: AMD
OpenGL Renderer: AMD Custom GPU 0932 (radeonsi, vangogh, LLVM 18.1.8, DRM 3.57, 6.5.0-valve22-1-neptune-65-g9a338ed8a75e)
OpenGL Version: 4.6 (Compatibility Profile) Mesa 24.2.7 (git-3900828265)
OpenGL Core: 4.6 (Core Profile) Mesa 24.2.7 (git-3900828265)
OpenGL ES: OpenGL ES 3.2 Mesa 24.2.7 (git-3900828265)
Vulkan Version: 1.3.290
Vulkan Drivers: AMD Custom GPU 0932 (RADV VANGOGH) (1.3.289)
```

**Comment by Plumblossomx** [comment]
I also encountered this problem

**Comment by ammarp** [comment]
Same issue for me. Controller works if we launch the game in lutris using desktop mode

**Comment by rybandrei2014** [comment]
Hey any solution to the problem. Happened to me after updating to 0.5.17. Is it possible to do downgrade?

**Comment by D3B453R** [comment]
> Hey any solution to the problem? Happened to me after updating to 0.5.17. Is it possible to do downgrade?

You could downgrade Lutris. One way to do this is to download Warehouse via Discover in Desktop mode on Steam Deck. Then search for Lutris, disable updates and select the previous version.

**Comment by ammarp** [comment]
> > Hey any solution to the problem? Happened to me after updating to 0.5.17. Is it possible to do downgrade?
> 
> You could downgrade Lutris. One way to do this is to download Warehouse via Discover in Desktop mode on Steam Deck. Then search for Lutris, disable updates and select the previous version.

Hi,

Thanks for the instruction. Did the controller start working after lutris downgrade ?

**Comment by ammarp** [comment]
Lutris just released update 0.5.18 and it has fixed the issue

**Comment by D3B453R** [comment]
> Lutris just released update 0.5.18 and it has fixed the issue 

I can confirm it, update 0.5.18 also fixed it for me as well. Time to play some Legacy of Kain Soul Reaver Remastered 😁

**Comment by legluondunet** [comment]
should be fixed now.
